runelite-client/src/main/java/net/runelite/client/plugins/inventorytags/InventoryTagsConfig.java

```diff
@@ -34,6 +35,7 @@
 {
 	String GROUP = "inventorytags";
 
+	@Alpha
```

Comment:
Does this work? Even if it does, the menu colors can't be alpha either so it would look weird when configuring ingame.